### PR TITLE
6.chatroom

### PR DESCRIPTION
**Pull request by ameyunaikgithub**

### CODE REVIEW

File: chat/public/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ Comment:<input type="text" id="Comment" />
+ <input type="submit" id="button" />
+ <div>
+ <pre id="chatid"/>
+ </div>
+ <script>
+const input =document.getElementById("button");
+input.onclick=()=>{
+  const name=document.getElementById("Name").value;
+  const comment=document.getElementById("Comment").value;
```
Comment (MadhuBeesetty) [inline chat/public/index.html]: use different variable names. it will confuse you with your id names in html file.

File: chat/chat.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const fs = require("fs");
+const app = express();
+app.listen(3614);
+app.use(express.static('public'));
+app.get('/anyurl', (req,res) =>{
+    fs.appendFile('/home/amey/garageScript/chat/public/data.txt',` name: ${req.query.Name} comment: ${req.query.Comment}\n`);
+    res.send("comment recorded and saved");
+    });
```
Comment (songz) [inline chat/chat.js]: indent issues.

File: chat/public/index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<title></title>
+</head>
+<body>
+ Name:<input type="text" id="Name" />
```
Comment (songz) [inline chat/public/index.html]: no indentation dude!

File: chat/chat.js (hunk 2 above)
Comment (ameyunaikgithub) [reply chat/chat.js]: solved

File: chat/public/index.html (hunk 3 above)
Comment (ameyunaikgithub) [reply chat/public/index.html]: solved

File: chat/chat.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const fs = require("fs");
+const app = express();
+app.listen(3614);
+app.use(express.static('public'));
+app.get('/anyurl', (req,res) =>{
```
Comment (songz) [inline chat/chat.js]: make url more reasonable. like... `/newChat`

File: chat/public/index.html [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    <div>
+      <pre id="chatid"/>
+ </div>
+ <script>
+const button =document.getElementById("button");
+button.onclick=()=>{
+  const name=document.getElementById("Name").value;
+  const comment=document.getElementById("Comment").value;
+  const xhttp=new XMLHttpRequest();
+  xhttp.open('GET', `/anyurl?Name=${name}&Comment=${comment}`);
```
Comment (saumya665) [inline chat/public/index.html]: change name of request

File: chat/chat.js (hunk 4 above)
Comment (ameyunaikgithub) [reply chat/chat.js]: hey song i changed and pushed the new files. Please check it

File: chat/public/index.html (hunk 5 above)
Comment (ameyunaikgithub) [reply chat/public/index.html]: hey i changed it ,cna you please review it

**Review by saumya665 — APPROVED** [review]

**Pull request merged**